Getting issue with fresh install

When I go to do meteor npm install

    npm WARN apollo-server@0.1.5 requires a peer of graphql@^0.5.0 || ^0.6.0 but none was installed.
    npm WARN express-widgetizer@0.5.11 requires a peer of graphql@^0.5.0 || ^0.6.0 but none was installed.
    npm WARN graphql-tools@0.6.0 requires a peer of graphql@^0.5.0 || ^0.6.0 but none was installed.

and when I run meteor I am getting the following error:

    Error: Cannot find module 'graphql/language' 

Any idea how to remedy?

--- a/client/main.js
+++ b/client/main.js
@@ -6,9 +6,6 @@ import ApolloClient from 'apollo-client';
 import { meteorClientConfig } from 'meteor/apollo';
 import { ApolloProvider } from 'react-apollo';
 
-import { registerGqlTag } from 'apollo-client/gql';
-registerGqlTag();
-
 import App from '/imports/ui/App';
 
 const client = new ApolloClient(meteorClientConfig());

--- a/imports/ui/App.js
+++ b/imports/ui/App.js
@@ -3,6 +3,7 @@ import { connect } from 'react-apollo';
 import { Meteor } from 'meteor/meteor';
 import { createContainer } from 'meteor/react-meteor-data';
 import { Accounts } from 'meteor/std:accounts-ui';
+import gql from 'graphql-tag';
 
 Accounts.ui.config({
   passwordSignupFields: 'USERNAME_ONLY'

--- a/package.json
+++ b/package.json
@@ -5,9 +5,11 @@
     "start": "meteor run"
   },
   "dependencies": {
-    "apollo-client": "^0.3.13",
+    "apollo-client": "^0.3.29",
     "apollo-server": "^0.1.5",
     "express": "^4.14.0",
+    "graphql": "^0.6.1",
+    "graphql-tag": "^0.1.9",
     "invariant": "^2.2.1",
     "meteor-node-stubs": "~0.2.0",
     "react": "^15.0.1",

